applications/launchpad/gui-react/src/components/TBot/DotsComponent/index.tsx

```diff
@@ -0,0 +1,33 @@
+import { useEffect, useRef } from 'react'
+import lottie from 'lottie-web'
+import dotsChatLottie from '../../../styles/lotties/tbot-dots-animation.json'
+import { StyledContainer, StyledDots } from './styles'
+
+/**
+ * @name ChatDots
+ */
+
+const ChatDots = () => {
+  const animation = useRef(null)
+  useEffect(() => {
+    if (animation.current) {
+      lottie.loadAnimation({
+        name: 'dotsAnimation',
+        container: animation.current,
+        renderer: 'svg',
+        loop: true,
+        autoplay: true,
+        animationData: dotsChatLottie,
+      })
+    }
+    return () => lottie.destroy()
```

Comment:
if destroy returns void, you can just do `return lottie.destroy` ?

Reply:
Done